<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfilesManagerDialog</class>
 <widget class="QDialog" name="ProfilesManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProfilesManagerDialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="profiles_manager.qrc">
    <normaloff>:/res/img/briefcase.ico</normaloff>:/res/img/briefcase.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabProfiles">
      <attribute name="title">
       <string>HTTP Ссылки</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayoutPF"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Удаленный доступ</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayoutRDP">
         <item>
          <widget class="QCheckBox" name="checkBoxEnableRdp">
           <property name="text">
            <string>Разрешить удаленный доступ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxEnableRdpUsers">
           <property name="text">
            <string>Разрешить удаленный доступ к сессиям пользователей</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="profiles_manager.qrc"/>
 </resources>
 <connections/>
</ui>
